Source/stores.cpp

```diff
@@ -83,9 +85,10 @@ void InitStores()
 {
 	int i;
 
-	pSTextBoxCels = LoadFileInMem("Data\\TextBox2.CEL", NULL);
-	pSPentSpn2Cels = LoadFileInMem("Data\\PentSpn2.CEL", NULL);
-	pSTextSlidCels = LoadFileInMem("Data\\TextSlid.CEL", NULL);

```

Comment:
stores.cpp isn't actually being build, we kept it for now in case we needed bits form it for some sort of shop analysis, but even if we do we sure aren't going to be loading these bits of UI graphics :)